tck/src/main/resources/tests/locationArray/input.graphql

```diff
@@ -0,0 +1,6 @@
+mutation logLocationForHero {
+  logLocationWithArray(name:"Starlord", coordinates:[-25.926804, 28.203392]) {
```

Comment:
Maybe use different coordinates from the other location test to ensure one isn't passing because the other passed.